core-web/libs/dotcms-models/src/lib/dot-device.model.ts

```diff
@@ -4,4 +4,5 @@ export interface DotDevice {
     cssWidth: string;
     name: string;
     inode: string;
+    icon?: string;
```

Comment:
Maybe we extend this just for the UI because this model if I am not mistaken match the API response.